content/diplomacy/gift-area-planets/plugin.js

```diff
@@ -0,0 +1,476 @@
+//
+// Gift Area Planets
+//
+// author: https://twitter.com/ddy_mainland
+//
+// Gift the planet(s) with colorful circle(s) on the map.
+// You can choose one of the receiver's planets and determine the address of just in the address.
+// For planet filter, this plugin supports level filter and have three modes:
+// set Single Planet, set Range Planet(s) and set All Planet(s).
+//
+//
+
+
+import { PlanetLevel, PlanetType, PlanetTypeNames } from
+	"https://cdn.skypack.dev/@darkforest_eth/types";
+
+import { html, render, useState } from
+	"https://unpkg.com/htm/preact/standalone.module.js";
```

Comment:
```suggestion
import { html, render, useState } from
	"https://unpkg.com/htm@3/preact/standalone.module.js";
```

Perhaps specify major preact version to use, to safeguard in case preact version 4 is not backwards compatible…